pkg/controller/controller.go

```diff
@@ -157,10 +157,12 @@ const (
 func (c *controller) reconcileBroker(broker *v1alpha1.Broker) {
 	glog.V(4).Infof("Processing Broker %v", broker.Name)
 
-	username, password, err := GetAuthCredentialsFromBroker(c.kubeClient, broker)
+	username, password, err := getAuthCredentialsFromBroker(c.kubeClient, broker)
 	if err != nil {
-		glog.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
-		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason, errorFetchingCatalogMessage)
+		err:= fmt.Errorf("Error getting broker auth credentials for broker %v: %v", broker.Name, err)
+		glog.Info(err)
+		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionFalse, errorFetchingCatalogReason,
```

Comment:
I think %q is good for the name since that appears in the middle of a sentence and the quotes help to make it stand out from the rest of the sentence. When the text appears by itself or separated (like after a colon) then I think %s should be ok